sphinx/search/__init__.py

```diff
@@ -553,3 +553,12 @@ def get_js_stemmer_code(self) -> str:
                     (base_js, language_js, self.lang.language_name))
         else:
             return self.lang.js_stemmer_code
+
+    def __getstate__(self):
```

Comment:
```suggestion
    def __getstate__(self) -> dict[str, Any]:
```

(and import Any, if needed)